dbt_project.yml

```
@@ -1,5 +1,5 @@
 name: 'social_media_reporting'
-version: '0.3.0'
+version: '0.4.0'
```

Oh wow seems like we did miss a bump. 

```suggestion
version: '0.5.0'
```